src/ray/common/function_descriptor.cc

```diff
@@ -65,6 +77,8 @@ FunctionDescriptor FunctionDescriptorBuilder::FromProto(rpc::FunctionDescriptor
     return ray::FunctionDescriptor(new ray::PythonFunctionDescriptor(std::move(message)));
   case ray::FunctionDescriptorType::kCppFunctionDescriptor:
     return ray::FunctionDescriptor(new ray::CppFunctionDescriptor(std::move(message)));
+  case ray::FunctionDescriptorType::kJuliaFunctionDescriptor:
```

Comment:
what's with the `k`?

Reply:
I think it's based on the enum `FunctionDescriptor` that's basically teh union type of all the known function descriptors...